release/tools/connector.xml

```diff
@@ -22,7 +22,7 @@
     <import file="../../bin/xml/ts.common.props.xml"/>
     
     <property name="deliverable.version" value="2.1"/>
-    <property name="deliverable.tck.version" value="2.1.0"/>
+    <property name="deliverable.tck.version" value="2.1.1"/>
```

Comment:
Hmm, guess we are looking to release new service releases with the GlassFish TCK script changes.

Reply:
From the [TCK process](https://jakarta.ee/committees/specification/tckprocess/):

> Process for releasing a point revision of a TCK
>
>The process for releasing a point revision of a TCK entails filing an issue in the [jakartaee/specifications repository with the following details](https://github.com/jakartaee/specifications):

> -   Link to the TCK release to be published.
> -   Updated TCK links in the specification’s _index.md file.
> -   A high-level description of what tests were excluded from the TCK and why.